starfish/pipeline/filter/gaussian_high_pass.py

```diff
@@ -0,0 +1,75 @@
+from functools import partial
+from typing import Callable
+
+import numpy as np
+
+from starfish.io import Stack
+from starfish.pipeline.filter.gaussian_low_pass import GaussianLowPass
+from ._base import FilterAlgorithmBase
+
+
+class GaussianHighPass(FilterAlgorithmBase):
+
+    def __init__(self, sigma: int, **kwargs):
+        """Gaussian high pass filter
+
+        Parameters
+        ----------
+        sigma : int
+            standard deviation of gaussian kernel
+
+        """
+        self.sigma = sigma
+
+    @classmethod
+    def get_algorithm_name(cls):
+        return "gaussian_high_pass"
+
+    @classmethod
+    def add_arguments(cls, group_parser):
+        group_parser.add_argument(
+            "--sigma", default=1, type=int, help="standard deviation of gaussian kernel")
```

Comment:
question: does it make sense that there is a default for the CLI but not for the API?

Reply:
Nope. This problem may exist in more places. I'll got through and check. There are a few ways to deal with this, None of the ones i'm thinking of are very appealing. 

1. Set the defaults in both places. This can lead to discrepancies with we're not careful. 
2. Set in the API only, and reference the defaults in the text of the CLI. We're still typing things in two places here, although the risk is that the defaults aren't what we say they are, if the CLI changes and the help strings lag behind
3. Set the default values as class properties and reference those in both the API and the CLI (in cli, using string formatting, + references to the defaults)

(3) seems best, but adds a hoop to jump through for new developers... I wonder if we could generate a script to bootstrap a new pipeline component 🙃